Automattic/wc-calypso-bridge
# Remove padding from title wrap

Fixes #136 

Removes the padding added to the left of the titles on screen sizes <783px.

### Screenshots
<img width="612" alt="screen shot 2018-11-02 at 11 31 59 am" src="https://user-images.githubusercontent.com/10561050/47928143-56a4cd80-de93-11e8-8a2f-f30c230d4b25.png">

### Testing
1.  Visit `/wp-admin/edit.php?post_type=shop_order` or any page with title wraps for h1 and h2.
2.  Confirm that no padding-left is added when screen is <783px.
